app/test/specs/institutionControllerSpec.js

```diff
@@ -67,83 +67,133 @@
         httpBackend.verifyNoOutstandingRequest();
     });
 
-    describe('Verify properties before initialize InstitutionController', function() {
+    describe('InstitutionController properties', function() {
 
-        it('Verify user', function() {
+        it('should exist a user and his name is Tiago', function() {
             expect(institutionCtrl.user.name).toEqual(tiago.name);
         });
 
-        it('Verify posts', function() {
+        it('should exist posts', function() {
             expect(institutionCtrl.posts).toEqual(posts);
         });
 
-        it('Verify members', function() {
+        it('should exist members', function() {
             expect(institutionCtrl.members).toEqual([tiago]);
         });
 
-        it('Verify followers', function() {
+        it('should exist followers', function() {
             expect(institutionCtrl.followers).toEqual([raoni]);
         });
 
-        it('Verify currentInstitution', function() {
+        it('should exist currentInstitution', function() {
             expect(institutionCtrl.current_institution).toEqual(splab);
         });
     });
     
-    describe('Test InstitutionController methods', function() {
-
-        it('Test follow method', function(done) {
-            spyOn(institutionService, 'getFollowers').and.callFake(function() {
-                return {
-                    then: function(callback) {
-                        return callback([raoni, tiago]);
-                    }
-                };
+    describe('InstitutionController methods', function() {
+
+        var promise;
+        
+        describe('follow()', function() {
+
+            beforeEach(function() {
+
+                spyOn(institutionService, 'getFollowers').and.callFake(function() {
+                    return {
+                        then: function(callback) {
+                            return callback([raoni, tiago]);
+                        }
+                    };
+                });
+                spyOn(institutionService, 'follow').and.callFake(function() {
+                    return {
+                        then: function(callback) {
+                            return callback();
+                        }
+                    };
+                });
+                spyOn(institutionCtrl.user, 'follow');
+                promise = institutionCtrl.follow();
             });
-            spyOn(institutionService, 'follow').and.callFake(function() {
-                return {
-                    then: function(callback) {
-                        return callback();
-                    }
-                };
+
+            it('should call institutionService.follow() ', function(done) {
+                promise.then(function() {
+                    expect(institutionService.follow).toHaveBeenCalledWith(splab.key);
+                    done();
```

Comment:
não usa o done() não, usa assim:

promise.then(function(){...}).should.notify(done);